usage/plugins/org.jboss.tools.usage/src/org/jboss/tools/usage/internal/branding/messages.properties

```diff
@@ -18,4 +18,4 @@ UsageReport_Checkbox_Text=Report usage of JBoss Tools to JBoss Tools team.
 UsageReport_HostName=jboss.org
 UsageReport_ExplanationPage=http://jboss.org/tools/usage
 
-GlobalUsageSettings_RemoteProps_URL=http://anonsvn.jboss.org/repos/jbosstools/workspace/usage/usage.properties
+GlobalUsageSettings_RemoteProps_URL=https://tools.jboss.org/usage/usage.properties
```

Comment:
Lets use http://tools.jboss.org/usage/usage.properties (HTTP instead of HTTPS) to avoid unnecessary redirection.